charts/rancher-gke-operator/1.1.100/Chart.yaml

```diff
@@ -5,4 +5,4 @@ home: https://github.com/rancher/gke-operator
 sources:
   - "https://github.com/rancher/gke-operator"
 version: 1.1.100
-appVersion: v1.1.1-rc6
+appVersion: v1.1.1-rc7
```

Comment:
Since this is 1.1.1, this is the version for 2.6. Why do we still need this one in system-charts if we transitioned to feature charts ?

Reply:
It's the forwardport of https://github.com/rancher/system-charts/pull/496. I thought that was the requirement?